src/Catty/ViewController/Download/StoreProjectDownloader.swift

```diff
@@ -24,11 +24,13 @@ protocol StoreProjectDownloaderProtocol {
     func fetchProjects(forType: ProjectType, offset: Int, completion: @escaping (StoreProjectCollection.StoreProjectCollectionText?, StoreProjectDownloaderError?) -> Void)
     func fetchSearchQuery(searchTerm: String, completion: @escaping (StoreProjectCollection.StoreProjectCollectionNumber?, StoreProjectDownloaderError?) -> Void)
     func fetchProjectDetails(for project: StoreProject, completion: @escaping (StoreProject?, StoreProjectDownloaderError?) -> Void)
+    func download(projectId: String, completion: @escaping (Data?, StoreProjectDownloaderError?) -> Void, progression: ((Float) -> Void)?)
 }
 
 final class StoreProjectDownloader: StoreProjectDownloaderProtocol {
 
     let session: URLSession
+    var observer: NSKeyValueObservation?
```

Comment:
Do we need to keep a reference to the KVO observation here? I do not think that this is necessary. If yes, please give it a better name.